lib/liquid-fixpoint/src/format.rs

```diff
@@ -318,13 +318,23 @@ impl<T: Types> fmt::Display for Expr<T> {
     }
 }
 
+fn hexify(n: u128, sz: usize) -> String {
+    let s = format!("{:x}", n);
+    let zeros = "0".repeat(sz - s.len());
+    format!("#x{}{}", zeros, s)
+}
+
 impl<T: Types> fmt::Display for Constant<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
             Constant::Numeral(i) => write!(f, "{}", i.display()),
             Constant::Decimal(r) => write!(f, "{}", r.display()),
             Constant::Boolean(b) => write!(f, "{b}"),
             Constant::String(s) => write!(f, "{}", s.display()),
+            Constant::BitVec(i, sz) => {
+                let str: u128 = i.display().to_string().parse().unwrap();
```

Comment:
yikes. I'll fix this later.

Reply:
haha yes I thought you might enjoy that odd workaround :-D